src/nrniv/netpar.cpp

```diff
@@ -836,31 +841,19 @@ static void mk_localgid_rep() {
 	// perfect hash functions, or even simple Vectors.
 	localmaps_ = new Gid2PreSyn*[nrnmpi_numprocs];
```

Comment:
Looks like `localmaps_` could probably be replaced with `std::vector<Gid2PreSyn>` to reduce indirection and manual memory management. But that's not critical for this PR.